spool_1/boolean_spec.rb

```diff
@@ -18,34 +18,34 @@
 
   describe "falsiness" do
     it "is false for false" do
-      expect(__).to be_false
+      expect(__).to be_falsey
```

Comment:
I think this is equally as clear as it used to be? Not sure what the comment would need to be like.